tracer/src/Datadog.Trace/Debugger/SpanCodeOrigin/SpanCodeOrigin.cs

```diff
@@ -27,6 +27,7 @@ internal class SpanCodeOrigin
 
         internal SpanCodeOrigin(DebuggerSettings settings)
         {
+            Log.Information("Initializing Code Origin for Spans");
```

Comment:
I'd rather keep these kinds of "everything ok" logs as debug if possible, they can add a surprising amount to startup latency